app/views/hiring_staff/vacancies/job_specification/show.html.haml

```diff
@@ -42,16 +42,9 @@
             Subject filter
         %span.govuk-hint#job-specification-form-subjects-hint
           =t('helpers.hint.job_specification_form.subjects_hint')
-        .checkboxes-with-scroll__filter.js-checkbox.display-none
-          %input.govuk-input#job-specification-form-subject-search{ type: 'text', 'aria-describedby': 'job-specification-form-subjects-hint', name: 'subject-search' }
 
-        = f.govuk_collection_check_boxes :subjects,
-          subject_options,
-          :first,
-          :first,
-          :last,
-          small: true,
-          classes: 'checkboxes-with-scroll subjects-options js-disabled-border'
+        %div{ class: 'govuk-!-margin-bottom-6' }
+          = render partial: 'shared/checkbox_group', locals: { :search => true, :options => subject_options, :form => 'job_specification_form[subjects]' }
```

Comment:
Could we do...

`= render 'shared/checkbox_group', search: true, f: f, attribute_name: :subjects, options: subject_options, value_method: :first, text_method: :first, hint_method: :last`